docs/reference/installation.rst

```diff
@@ -56,44 +73,58 @@ files:
         );
     }
 
-.. note::
-
-    If you already have installed a Sonata dependency, you may ignore the step
-    on the modification of the ``autoload.php`` file.
-
 Configuration
 -------------
-Here is the configuration you will need to add:
+
+.. note::
+    If you are not using Symfony Flex, all configuration in this section should
+    be added to ``app/config/config.yml``.
+
+SonataCommentBundle Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. code-block:: yaml
 
-    # app/config/config.yml
+    # config/packages/sonata.yaml
 
     sonata_comment:
         manager_type: orm # can be 'orm' or 'mongodb'
         class:
-            comment: Application\Sonata\CommentBundle\Entity\Comment # This is an optional value
-            thread: Application\Sonata\CommentBundle\Entity\Thread   # This is an optional value
+            comment: App\Application\Sonata\CommentBundle\Entity\Comment # This is an optional value
+            thread: App\Application\Sonata\CommentBundle\Entity\Thread   # This is an optional value
 
-Doctrine Configuration
-~~~~~~~~~~~~~~~~~~~~~~
-Then, add these bundles in the config mapping definition (or enable `auto_mapping <http://symfony.com/doc/2.0/reference/configuration/doctrine.html#configuration-overview>`_):
+.. note::
+    If you are not using Symfony Flex, add classes without the ``App\``
```

Comment:
WDYT about moving this note 📝 to an extra file and include it?

Reply:
Not sure if it is worth it, I can do it but some PRs were already merged where the notes were duplicated.